products/caregivers/Improve Facility Selection/10-10 Health Apps-10-10CG-[date].md

```diff
@@ -2,71 +2,90 @@
+  - Describe new or modified databases, tables or columns
+    - No Changes. The facility that is ultimately returned is used to pass the facility id as part of the 1010CG Form submission.
+  - Describe indexes and constraints
+    - No Changes
+  - Identify PII and PHI and where and how it will be stored, processed, expired and deleted
+    - No Changes
 - Libraries and dependencies
-    + List new or updated dependencies
+  - List new or updated dependencies
 - Metrics, logging, observability, alerting
-    + Identify key areas to monitor
+  - Identify key areas to monitor
 - Infrastructure and network changes
-    + List any changes or additions
+  - List any changes or additions
 - Test strategy
-    + Describe automated, manual and user acceptance test strategy
-    + Describe required test data and test user accounts
-         + This is an aunauthenticated form, meaning that a user does not need to be signed in to complete and submit it.  We do not use any data from an authenticated user to pre-fill this form or provide any statuses.
+  - Describe automated, manual and user acceptance test strategy
+  - Describe required test data and test user accounts
+    - This is an aunauthenticated form, meaning that a user does not need to be signed in to complete and submit it. We do not use any data from an authenticated user to pre-fill this form or provide any statuses.
 - Rollout plan
-    + List scope of any feature flags
-         + Feature flag: caregiver_use_facilities_API
-         + Feture flag description: Allow list of caregiver facilites to be fetched by way of the Facilities API
-    + Identify other teams to coordinate with
-    + Describe rollback plan
-         + The PM and Data Analyst will monitor analytics. If there is a spike in errors or unexpected behavior, the feature toggle will be used to disable the feature in production and issue triage will begin.
+  - List scope of any feature flags
+    - Feature flag: caregiver_use_facilities_API
+    - Feture flag description: Allow list of caregiver facilites to be fetched by way of the Facilities API
+  - Identify other teams to coordinate with
+    - None
+  - Describe rollback plan
+    - The PM and Data Analyst will monitor analytics. If there is a spike in errors or unexpected behavior, the feature toggle will be used to disable the feature in production and issue triage will begin.
 - Internal administration tasks
-    + What maintenance or administration tasks do you anticipate will need to be performed periodically?
-    + Describe how you intend for these tasks to be performed (e.g., through an internal web page, through terminal access, etc.).
+  - What maintenance or administration tasks do you anticipate will need to be performed periodically?
+    - If either of the mapbox api or Lighthouse facilities api change we may need to do something. Those are not expected, and are used a few other places in `vets-website` and `vets-api` so these changes are not unique.
```

Comment:
Yeah, maybe not something to call out since that is really how api versioning works. It looks like we use the mapbox sdk on the frontend so that would stay consistent.